### ISSUE TITLE
Tagbar resets buffer position when previewing

### ISSUE THREAD

**Issue by TheSlovakGuy** [majutsushi/tagbar#460]
So the thing is that everytime I try to look at tag in preview window, it resets the buffer to first row.   
![Before](https://user-images.githubusercontent.com/10329816/33526997-74985556-d849-11e7-89fa-9ddf33129697.png)
![After](https://user-images.githubusercontent.com/10329816/33527001-7d7cc274-d849-11e7-945c-a16f245672ef.png)
I don't know if this is supposed to happen, but it's pretty anoying. Any way to fix this?

EDIT: This might be useful:  
Basically what I do is jump to tag (via return) and then try to display something in preview. That's when the buffer resets.

**Comment by majutsushi** [comment]
What do you mean exactly by "try to display something in preview"? How do you do that? I assume you're not using the `g:tagbar_autopreview` option? If not I'm not sure this is actually related to Tagbar.

**Comment by TheSlovakGuy** [comment]
Right. Sorry for the confusion. It's the `<P>` button (`" P: Show tag in preview window`).

**Comment by majutsushi** [comment]
Hmm, I can't reproduce this. Maybe some autocommands or another plugin is interfering?

**Comment by TheSlovakGuy** [comment]
Ok. I tried fiddling again with my vimrc and it seems that one of the plugins might be causing interference. I will post which if I find it...

**Comment by TheSlovakGuy** [comment]
And found it... It was `guns/vim-sexp`. Sorry for bothering.

**Comment by alerque** [comment]
It sounds like this got resolved and it wasn't our bug, so I'm going to close the issue. Feel free to comment if anything is left unresolved.

I will also cross-link #486 because it's possible what people there are experiencing is related.
